packages/editor-ui/src/components/Node.vue

```diff
@@ -106,9 +113,31 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 
 			return workflowResultDataNode.length;
 		},
+		nodeOffsetPosition() {
```

Comment:
might be helpful to rename this to `canvasOffsetPosition`.. it's pretty confusing in this context

Reply:
Refactored